### DISCUSSION TITLE
Coding challenge with windows and wsl ubuntu simulator is not working.

### DISCUSSION THREAD

**Discussion by esrefbozkurt** [DevSkillsHQ/discusssions#61] [Help]
Unfortunately I'm not able to start an emulator. Please help setup the challenge properly so that i can work on the android studio emulator.

**Comment by fenske** [comment]
Hi @esrefbozkurt. Did this get resolved in the end?

**Reply by esrefbozkurt** [reply]
Hi @fenske! Yes. I described the whole setup process or the issues I had in the description of the pull request. After hours of trial and error, I decided to install and setup everything from scratch on a friend’s MacBook, where I was able to connect the iOS Simulator. And continued developing on the new machine. Thanks a lot!
